w3af/plugins/grep/analyze_cookies.py

```diff
@@ -204,13 +204,16 @@ def _http_only(self, request, response, cookie_obj,
         :return: None
         """
         if not self.HTTPONLY_RE.search(cookie_header_value):
-            
             vuln_severity = severity.MEDIUM if fingerprinted else severity.LOW
-            desc = 'A cookie without the HttpOnly flag was sent when ' \
+            keys = ''
+            for key in cookie_obj.keys(): 
+                keys += key + ' '
+            keys = keys.strip()
+            desc = 'Cookie(s) (%s) without the HttpOnly flag was sent when ' \
```

Comment:
I believe that we should always have descriptions saying `Cookie "foobar" was sent without httponly", no `(s)` should be present.